### PR TITLE
🎨 moving projects between workspaces

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
- 🎨 moving projects between workspaces

## Related issue/s
- https://github.com/ITISFoundation/osparc-issues/issues/716
- https://github.com/ITISFoundation/osparc-issues/issues/1606

## How to test
`test_workspaces__moving_projects_between_workspaces.py`

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/web/server/src/simcore_service_webserver/projects/_groups_db.py [hunk 1]
```diff
@@ -190,3 +190,15 @@ async def delete_project_group(
                 & (project_to_groups.c.gid == group_id)
             )
         )
+
+
+async def delete_all_project_groups(
```
Comment (sanderegg) [inline services/web/server/src/simcore_service_webserver/projects/_groups_db.py:195]: this name is so confusing ;) why already is it not called project access rights?

File: services/web/server/src/simcore_service_webserver/projects/_workspaces_api.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    project_api: ProjectDBAPI = app[APP_PROJECT_DBAPI]
+
+    # 1. User needs to have delete permission on project
+    project_access_rights = await get_user_project_access_rights(
+        app, project_id=project_id, user_id=user_id, product_name=product_name
+    )
+    if project_access_rights.delete is False:
+        raise ProjectInvalidRightsError(user_id=user_id, project_uuid=project_id)
+
+    # 2. User needs to have write permission on workspace
```
Comment (sanderegg) [inline services/web/server/src/simcore_service_webserver/projects/_workspaces_api.py:37]: ... will say nothing

**Review by sanderegg — APPROVED** [review]
thanks!

File: services/web/server/src/simcore_service_webserver/projects/_workspaces_api.py (hunk 2 above)
Comment (matusdrobuliak66) [reply services/web/server/src/simcore_service_webserver/projects/_workspaces_api.py:37]: I know still in my todo list, I will do it ones everywhere and properly ;)

**Pull request merged**